dashboard/assets/components/Main.jsx

```diff
@@ -0,0 +1,158 @@
+                {
+                    React.Children.map(this.props.children, child => (
+                        <Grid item xs={child.props.xs}>
+                            <ResponsiveContainer width="100%" height={child.props.height}>
+                                {child}
+                            </ResponsiveContainer>
+                        </Grid>
+                    ))
+                }
+            </Grid>
+        );
+    }
+}
+
+ChartGrid.propTypes = {
+    spacing: PropTypes.number.isRequired,
+};
+
+// ContentSwitch chooses and renders the proper page content.
+class ContentSwitch extends Component {
+    render() {
+        switch(this.props.active) {
+            case TAGS.home.id:
+                return (
+                    <ChartGrid spacing={24}>
+                        <AreaChart xs={6} height={300} data={this.props.memory}>
+                            <YAxis />
+                            <Area type="monotone" dataKey="memory" stroke="#8884d8" fill="#8884d8" />
+                        </AreaChart>
+                        <LineChart xs={6} height={300} data={this.props.traffic}>
+                            <Line type="monotone" dataKey="traffic" dot={false} />
+                        </LineChart>
+                        <LineChart xs={6} height={300} data={this.props.memory}>
+                            <YAxis />
+                            <CartesianGrid stroke="#eee" strokeDasharray="5 5" />
+                            <Line type="monotone" dataKey="memory" stroke="#8884d8" dot={false} />
+                        </LineChart>
+                        <AreaChart xs={6} height={300} data={this.props.traffic}>
+                            <CartesianGrid stroke="#eee" strokeDasharray="5 5" vertical={false} />
+                            <Area type="monotone" dataKey="traffic" />
+                        </AreaChart>
+                    </ChartGrid>
+                );
+            case TAGS.logs.id:
+                return <div>{this.props.logs.map((log, index) => <div key={index}>{log}</div>)}</div>;
+            case TAGS.networking.id:
+                // Only for testing.
+                return (
+                    <Grid container spacing={24}>
+                        <Grid item xs={6}>
+                            <ResponsiveContainer width="100%" height={300}>
+                                <LineChart data={this.props.traffic}>
+                                    <Line type="monotone" dataKey="traffic" dot={false} />
+                                </LineChart>
+                            </ResponsiveContainer>
+                        </Grid>
+                        <Grid item xs={6}>
+                            <ResponsiveContainer width="100%" height={300}>
+                                <AreaChart data={this.props.memory}>
+                                    <YAxis />
+                                    <Area type="monotone" dataKey="memory" stroke="#8884d8" fill="#8884d8" />
```

Comment:
Please use theme colors everywhere instead of custom ones so they can be easily swapped out with the theme.